full/src/main/java/apoc/custom/CypherProceduresHandler.java

```diff
@@ -621,9 +619,11 @@ public void removeProcedure(String name) {
 
     public void removeFunction(String name) {
         withSystemDb(tx -> {
+            Map<String, Object> fullName = getFullName(name);
             tx.findNodes(SystemLabels.ApocCypherProcedures,
                     SystemPropertyKeys.database.name(), api.databaseName(),
-                    SystemPropertyKeys.name.name(), name
+                    SystemPropertyKeys.name.name(), fullName.get("name"),
```

Comment:
a cosa servono le variabili in più?

Reply:
Perché nel caso di una procedura con i punti, ad esempio `custom.a.b.c` quando cancello una procedura per cancellare quella corretta mi serve sia il nome, come prima (quindi `"c"`) che il prefisso della procedura salvato nello `storeFunction(..)` (quindi `["custom", "a", "b"]`)